C-Sharp/VectorDistanceCalculation/PE11VectorDistanceCalculation/Menu.cs

```diff
@@ -10,23 +10,31 @@ class Menu
     {
         private static int selection;
         private static int[] validSelections = { 1, 2, 3 };
+        //Have you guys gone over unit tests?
+        //I've seen a few bugs in your code but unit tests will surface those bugs
         public static void Run()
         {
             displayWelcomeMessage();
             
             while (true)
             {
-                selection = getValidUserInput();
+                selection = getValidUserInput();//start all letters with capital letter
+                //rule of thumb but all switch statesments should have a a default
+                //even though your switch statement is ok now but what happens if future you or another dev adds another valid input?
                 switch (selection)
                 {
                     case 1:
-                        Point2DCollection p2c = new Point2DCollection();
+                        Point2DCollection p2c = new Point2DCollection();//var
                         Console.WriteLine(p2c);
+                        //use string verbatim/interpolation here
+                        //https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/
+                        //p2c.TwoClosestPoints[0] models/objects should be separated out from this type of logic
```

Comment:
Just for clarification, are you recommending to wrap it in a separate method?

Reply:
Nope. Try var fooString = @$"foo string 
contents";

The @ removes the need for "+" concatenation

Reply:
Got it!